### PR TITLE
Red hat build

### PR DESCRIPTION
**Pull request by macumber**

### CODE REVIEW

File: dependencies/Boost/gcc_5_no_cxx11.patch [hunk 1]
```diff
@@ -0,0 +1,37 @@
+From eec808554936ae068b23df07ab54d4dc6302a695 Mon Sep 17 00:00:00 2001
```
Comment (macumber) [inline dependencies/Boost/gcc_5_no_cxx11.patch]: @lefticus I applied this patch to get Boost 1.55 to build with gcc 5.2 on RedHat 6.6.  Do you see any issues applying this patch on Ubutnu too?

@axelstudios @kbenne FYI

File: openstudiocore/src/utilities/geometry/PointLatLon.cpp [hunk 2]
```diff
@@ -20,9 +20,9 @@
 #include "PointLatLon.hpp"
 #include "Point3d.hpp"
 
-#include "../../GeographicLib/Geodesic.hpp"
-#include "../../GeographicLib/LocalCartesian.hpp"
-#include "../../GeographicLib/UTMUPS.hpp"
+#include <GeographicLib/Geodesic.hpp>
```
Comment (macumber) [inline openstudiocore/src/utilities/geometry/PointLatLon.cpp]: @axelstudios GeographicLib is built and installed in the build directory, I'm not sure how this was working anywhere

@kbenne FYI

File: dependencies/Boost/gcc_5_no_cxx11.patch (hunk 1 above)
Comment (lefticus) [inline dependencies/Boost/gcc_5_no_cxx11.patch]: One question: why was this necessary? gcc 5.2 supports C++11/14/17

And to answer your question, I'd strongly recommend against applying where not needed because it could have a significant impact on compile times (variadic templates are much faster than the alternative).

Comment (macumber) [inline dependencies/Boost/gcc_5_no_cxx11.patch]: [This is the error ](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=197594) I was seeing

When I tried to apply the patch in that first report it failed for some reason that I can't remember.  I found the other patch [here](https://github.com/bitcoin/bitcoin/pull/6280/files) and it fixed my compile errors.  I think we need to update boost anyway pretty soon.  If compile time is the only downside then I'd be inclined to merge this and then delete it when we upgrade boost.

File: openstudiocore/src/utilities/geometry/PointLatLon.cpp (hunk 2 above)
Comment (macumber) [inline openstudiocore/src/utilities/geometry/PointLatLon.cpp]: @evanweaver

Comment (axelstudios) [inline openstudiocore/src/utilities/geometry/PointLatLon.cpp]: ¯\_(ツ)_/¯

**Comment by nrel-bot-3** [comment]
@macumber @lgentile it has been 7 days since this pull request was last updated.
